EpicLoot/config/magiceffects.json

```diff
@@ -2240,6 +2478,54 @@
         "$mod_epicloot_me_addbloodmagicskill_suffix1"
       ]
     },
+    {
+      "Type" : "ModifySummonHealth",
```

Comment:
For summons defaults this may be too high. I would make them similar to the magic effects for the player where mythic does not go higher than 18% for damage (see "ModifyDamage" effect). Health for the player is a flat increase so I would use similar values to the damage for a health percentage.

Reply:
Fixed in: https://github.com/OrianaVenture/Randy_Vapok_ValheimMods/pull/10/commits/dbb76034c2c58c647d46e3a77e6f6fe797e26715